### PR TITLE
[OTTO-78] Add php:cos:update command

### PR DESCRIPTION
**Pull request by greg-1-anderson**

### CODE REVIEW

File: src/Cli/PhpCommands.php [hunk 1]
```diff
@@ -32,6 +32,104 @@ protected function phpIsEOL($work_dir, $version)
...
+        $this->logger->notice('Using {branch}', ['branch' => $branch]);
+        $cos_php
+            ->createBranch($branch, 'master', true)
+            ->add('PHP_VERSIONS')
+            ->commit($message)
+            ->push()
+            ->pr($message);
+
+        // Once we create a new PR, we can close the existing PRs.
+        $api->prClose($cos_php->org(), $cos_php->project(), $prs);
```
Comment (pwtyler) [inline src/Cli/PhpCommands.php:114]: Would be nice to link to new PR when closing old ones if that's straightforward.

File: updatinate.yml [hunk 2]
```diff
@@ -20,14 +20,13 @@ php-net:
   download-url: http://php.net/distributions/php-{version}.tar.gz
 
 projects:
+  cos-php:
+    repo: git@github.com:pantheon-systems/cos-runtime-php.git
+    path: ${working-copy-path}/cos-php
+    #fork: git@github.com:pantheon-circleci/cos-runtime-php.git
```
Comment (pwtyler) [inline updatinate.yml:26]: should this commented line be removed before merge?

File: src/Cli/PhpCommands.php (hunk 1 above)
Comment (greg-1-anderson) [reply src/Cli/PhpCommands.php:114]: That's a good suggestion, but there are a number of places where we close old PRs in favor of a new one. I think I'll leave this as follow-on work, so I can do it consistently without putting our sprint goals at risk.

File: updatinate.yml (hunk 2 above)
Comment (greg-1-anderson) [reply updatinate.yml:26]: We decided to give the pantheon-upstream user write access to the cos-runtime-php repo, so we won't be using this for now.

I do wonder, though, if it wouldn't be better for ALL of the PRs to use "fork", and we currently have none using it. I think I'll keep this around for (light) documentation.

File: src/Cli/PhpCommands.php (hunk 1 above)
Comment (greg-1-anderson) [reply src/Cli/PhpCommands.php:114]: Created [OTTO-334](https://getpantheon.atlassian.net/browse/OTTO-334)

**Pull request merged**